src/main/java/lotto/domain/Lottos.java

```diff
@@ -24,19 +25,18 @@ private int purchasableLottoCount(int value) {
         return value / Lotto.LOTTO_PRICE;
     }
 
-    public int totalWinningAmount(String winningNumber) {
-        WinningAmount winningAmount = new WinningAmount();
+    public Long totalWinningAmount(WinningNumbersDTO winningNumbersDTO) {
+        Amount amount = new Amount();
         for (Lotto lotto : this.lottos) {
-            winningAmount.add(lotto.winningInfo(winningNumber).getWinningAmount());
+            amount.add(lotto.winningInfo(winningNumbersDTO).getWinningAmount());
```

Comment:
```suggestion
            Lotto lotto1 = Lotto.of(winningNumbersDTO.getWinningNumbers2());
            LottoNumber bonus = LottoNumber.of(winningNumbersDTO.getBonusNumber());
            int a = lotto.correctCount(lotto1);
            int b = lotto.bonusCorrectCount(bonus);
            WinningInfoDTO result = new WinningInfoDTO(a, b, new Amount(Winning.winningAmount(a, b)));
            amount.add(result.getWinningAmount());
```

저는 최대한 객체를 만들고 상태를 가지는 객체에게 메시지를 전달해볼수 있도록 할것 같네요~
위 코드는 저의 관점을 예시로 표현해보았습니다~
참고해주세요~

> 변수명은 임의로 작성했습니다.